groups/restrictions.yaml

```diff
@@ -27,6 +27,7 @@ restrictions:
   - path: "sig-apps/groups.yaml"
     allowedGroups:
       - "^k8s-infra-staging-examples@kubernetes.io$"
+      - "^k8s-infra-staging-jobset@kubernetes.io$"
```

Comment:
This might be the file preventing the merge

Reply:
any idea how to address it? are we not supposed to this?

Reply:
actually, Bob pointed out that the culprit is [k8s.gcr.io/manifests/k8s-staging-jobset/promoter-manifest.yaml](https://github.com/kubernetes/k8s.io/pull/5144/files#diff-5ea082290b72efae9bf40dd30f2d1e4ed8a079fa440cbacb57a332fbb360aed0) which should move to registry.k8s.io/...